v0/destinations/tiktok_ads/data/TikTokTrack.json

```diff
@@ -0,0 +1,47 @@
+[
+  {
+    "destKey": "event_id",
+    "sourceKeys": "properties.eventId",
+    "required": false
+  },
+  {
+    "destKey": "timestamp",
+    "sourceKeys": "timestamp",
+    "required": false
+  },
+  {
+    "destKey": "test_event_code",
+    "sourceKeys": "properties.testEventCode",
+    "required": false
+  },
+  {
+    "destKey": "properties",
```

Comment:
Lets segregate this to individual mappings

Reply:
segregated